core/templates/pages/contributor-dashboard-page/modal-templates/translation-modal.directive.html

```diff
@@ -99,6 +106,12 @@
     margin: 15px 0;
     padding: 20px;
   }
+  .oppia-translation-tips-container {
+    background-color: #ccc;
```

Comment:
Can we make the background color slightly lighter? I think the light gray used for the text box editing toolbox would work.

Reply:
Done, New color posted below

Reply:
<img width="656" alt="Screen Shot 2021-04-01 at 9 35 11 AM" src="https://user-images.githubusercontent.com/35385366/113325702-a048ab00-92cd-11eb-8b9b-782fc5b4ff64.png">